src/panels/lovelace/elements/hui-icon-element.ts

```diff
@@ -8,10 +8,16 @@ import { actionHandler } from "../common/directives/action-handler-directive";
 import { handleAction } from "../common/handle-action";
 import { hasAction } from "../common/has-action";
 import { IconElementConfig, LovelaceElement } from "./types";
+import { LovelacePictureElementEditor } from "../types";
 import { ActionHandlerEvent } from "../../../data/lovelace/action_handler";
 
 @customElement("hui-icon-element")
 export class HuiIconElement extends LitElement implements LovelaceElement {
+  public static async getConfigElement(): Promise<LovelacePictureElementEditor> {
```

Comment:
We should add support for `getStubConfig` so the elements are not created with an invalid config